website/docs/docs/collaborate/govern/model-access.md

```diff
@@ -163,16 +163,38 @@ Of course, dbt can facilitate this by means of [the `grants` config](/reference/
 
 As we continue to develop multi-project collaboration, `access: public` will mean that other teams are allowed to start taking a dependency on that model. This assumes that they've requested, and you've granted them access, to select from the underlying dataset.
 
-### What about referencing models from a package?
+### How do I ref a model from another project?
 
-For historical reasons, it is possible to `ref` a protected model from another project, _if that protected model is installed as a package_. This is useful for packages containing models for a common data source; you can install the package as source code, and run the models as if they were your own.
+<VersionBlock lastVersion="1.5">
+
+In dbt Core v1.5, the only way to reference a model from another project is by installing that project as a package. There is no way to enforce access distinctions between `protected` and `public` models.
+
+This changes in v1.6. Select it from the dropdown above.
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.6">
+
+It is possible to `ref` a model from another project in two ways:
+1. As a "project" dependency, via "cross-project `ref`" (a feature of dbt Cloud Enterprise)
+2. As a "package" dependency, whereby all source code from that project is installed into your own
+
+See ["Project Dependencies"](project-dependencies) for an explanation of the advantages of each approach.
+
+#### How do I restrict access to models defined in a package?
+
+The maintainer of a project that can be installed as a package may choose whether to restrict other models' `ref` access to only its public models. The package maintainer does this by setting a `restrict_access` config to `True`. By default, the config is `False`, meaning that:
+- Models in the package with `access: protected` may be referenced by models in the root project, as if they were defined in the same project
+- Models in the package with `access: private` may be referenced by models in the root project, so long as they also have the same `group` config
+
+When `restrict_access: True`, any `ref` from outside the package to a protected or private model in that package will fail. Only models with `access: public` may be accessed outside the package.
```

Comment:
This was actually implemented as [restrict-access](https://github.com/dbt-labs/dbt-core/pull/7962/files#diff-2c495a0925e5096258f6e87aa1784f8139a23c940601d816eaa8db5e5ad9975aR176) for consistency with other dbt_project.yml-level configs.

Reply:
Ah, yep, makes sense. It would be nice if we were globally consistent about snake versus kebab casing (snake should really win)